File: src/main/java/fr/inativ/dojo/Application.java
package fr.inativ.dojo;

public class Application {

    public static void main(String[] args) {
        Application app = new Application();
        Integer first = Integer.valueOf(args[0]);
        Integer second = Integer.valueOf(args[1]);
        System.out.println("Resultat = " + app.add(first, second));
    }

    public Integer add(Integer n1, Integer n2) {
        return n1 + n2;
    }
}

File: src/main/kotlin/Application.kt
fun main(args: Array<String>) {
    val n1 = Integer.valueOf(args[0])
    val n2 = Integer.valueOf(args[1])
    println("Resultat ${add(n1, n2)}")
}

fun add(n1: Int, n2: Int) = n1 + n2